### PR TITLE
Update dns_resolver-type.h

### PR DESCRIPTION
**Pull request by KotwalNikhil**
updated line 5 updated

### CODE REVIEW

**Comment by admsev** [comment]
@gooroodev please review

**Comment by gooroodev** [comment]
Thank you for tagging me, @admsev!

{mistakes: 1, typos: 1, security: 0, performance: 0, code_styling: 0, others: 0}

1. **Summary of changes:**
   - The pull request introduces a typo in the declaration of the `key_type_dns_resolver` variable. Specifically, the string `vuuyf` was appended to the end of the line.

2. **Issues, bugs, or typos:**
   - **Typo in code:**
     ```diff
     -extern struct key_type key_type_dns_resolver;
     +extern struct key_type key_type_dns_resolver;vuuyf
     ```
     **Proposed improvement:**
     ```diff
     -extern struct key_type key_type_dns_resolver;vuuyf
     +extern struct key_type key_type_dns_resolver;
     ```

3. **General review of the code quality and style:**
   - The original code is well-formatted and adheres to common C coding standards. The inclusion of the SPDX license identifier is a good practice for open-source projects.
   - The introduced change, however, seems to be an accidental typo which should be corrected.

In conclusion, the pull request currently introduces an error rather than an improvement. The typo should be removed to maintain the integrity of the code.

Yours, [Gooroo.dev](https://github.com/apps/gooroo-dev). Feel free to react or reply with your feedback!
